packages/@n8n/db/src/migrations/common/1762177736257-AddWorkflowDescriptionColumn.ts

```diff
@@ -0,0 +1,20 @@
+import type { MigrationContext, ReversibleMigration } from '../migration-types';
+
+const columnName = 'description';
+const tableName = 'workflow_entity';
+
+export class AddWorkflowDescriptionColumn1762177736257 implements ReversibleMigration {
+	async up({ escape, runQuery }: MigrationContext) {
+		const escapedTableName = escape.tableName(tableName);
+		const escapedColumnName = escape.columnName(columnName);
+
+		await runQuery(`ALTER TABLE ${escapedTableName} ADD COLUMN ${escapedColumnName} TEXT`);
```

Comment:
We can use the add column helper. No need to do the query ourselves.

<img width="992" height="352" alt="Image" src="https://github.com/user-attachments/assets/387d28be-d786-40b4-ac2c-d40d926c9b86" />